CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 * Avoid to process again challenges that are already validated
   when a certificate is issued.
+* If possible, Certbot uses built-in support for OCSP from recent cryptography
+  versions instead of OpenSSL binary: as a consequence Certbot does not need
+  OpenSSL binary to be installed anymore if cryptography>=2.5 is installed.
```

Comment:
nit: `s/OpenSSL binary/the OpenSSL binary/g`

There are two occurrences of this in this changelog entry. Without the "the" it totally makes sense, but with the "the", it sounds a little more natural/correct in my opinion.

Reply:
Corrected!